util/perl/OpenSSL/ParseC.pm

```diff
@@ -816,6 +816,7 @@ sub parse {
 
     foreach my $line (@_) {
         # split tries to be smart when a string ends with the thing we split on
+        $line =~ s/\r\n/\n/; # gracefully handle DOS-style end-of-line
```

Comment:
This doesn't seem right...  But, the bug would suggest that the split at (original) line 825 (a few lines down) doesn't work as intended, i.e. that `$/` is less flexible than desired.  So, what happens if you make this replacement on line 825 instead?

``` diff
-        my @lines = map { ( undef, $_ ) } split $/, $line;
+        my @lines = map { ( undef, $_ ) } split m|\R|, $line;
```

Reply:
`\R` documentation here: https://perldoc.perl.org/perlrebackslash#%5CR

Reply:
Nice - this works as well and is more elegant.
Changed.